extensions/msal-node-extensions/src/persistence/FilePersistenceWithDataProtection.ts

```diff
@@ -39,28 +39,39 @@ export class FilePersistenceWithDataProtection implements IPersistence {
     }
 
     public async save(contents: string): Promise<void> {
+        const TIMESTAMP_LABEL = "save -> FilePersistenceWithDataProtection";
         try {
+            // eslint-disable-next-line no-console
+            console.time(TIMESTAMP_LABEL);
             const encryptedContents = Dpapi.protectData(
                 Buffer.from(contents, "utf-8"),
                 this.optionalEntropy,
                 this.scope.toString());
             await this.filePersistence.saveBuffer(encryptedContents);
+            // eslint-disable-next-line no-console
+            console.timeEnd(TIMESTAMP_LABEL);
```

Comment:
I am curious about the need to include these timings in the extension library itself, can you provide some details on that?

Reply:
I added the timings temporarily to be able to get the timings across the various environments easily. I don't plan to leave this piece of code there.

Reply:
Also converted the pull request to a draft till I clean it up.